bip-0052.mediawiki

```diff
@@ -0,0 +1,302 @@
+<pre>
+  BIP: 52
```

Comment:
Missing Layer header

```
  Layer: Consensus (hard fork)
```

Note this will affect the first line in the README entry

Reply:
Thank you for clarifications: c689a92e353447ce81e6c50b6243adec0a4f4522